xml/schema/Event/Event.xml

```diff
@@ -9,10 +9,15 @@
   <field>
     <name>id</name>
     <type>int unsigned</type>
-    <title>Event</title>
+    <title>Event ID</title>
     <required>true</required>
     <comment>Event</comment>
     <add>1.7</add>
+   <pseudoconstant>
+      <table>civicrm_event</table>
+      <keyColumn>id</keyColumn>
+      <labelColumn>title</labelColumn>
+    </pseudoconstant>
```

Comment:
@monishdeb what was the reason for this change? It breaks a few things e.g. in #5675 and imo is not correct use of `<pseudoconstant>` - it is redundant to declare an entity as being the pseudoconstant of itself.